src/entity/notifications/NotificationV2.ts

```diff
@@ -52,3 +62,42 @@ export class NotificationV2 {
   @Column({ type: 'uuid', array: true, default: null })
   avatars: string[];
 }
+
+export const getNotificationV2AndChildren = (
```

Comment:
I always wonder if we should include this in the entity file actually?
(Feels a bit messy)

Reply:
Agreed. Just stuck to existing practices. We have a similar function in `Notification`. Happy to move it elsewhere

Reply:
Maybe let's set the right tone and move it away from the entity folder.
`src/notification/common` might even work?

Reply:
Sure let me do that